src/app/components/mnoe-dashboard-templates-list/mnoe-dashboard-templates-list.html

```diff
@@ -0,0 +1,77 @@
+<mno-widget icon="fa-bar-chart" is-loading="$ctrl.dashboardTemplates.loading">
+  <mno-widget-header>
+    &nbsp;{{ 'mnoe_admin_panel.dashboard.dashboard_templates.widget.list.title' | translate}}
+  </mno-widget-header>
+  <mno-widget-body class="no-padding">
+    <div>
+      <table st-pipe="$ctrl.callServer" class="table table-striped" st-table="$ctrl.dasboardTemplates.list">
+        <thead>
+          <tr>
+            <th st-sort="name" class="col-lg-4" translate>mnoe_admin_panel.dashboard.dashboard_templates.widget.list.template_name</th>
+            <th class="col-lg-2" translate>mnoe_admin_panel.dashboard.dashboard_templates.widget.list.status</th>
+            <th class="col-lg-2" translate>mnoe_admin_panel.dashboard.dashboard_templates.widget.list.created</th>
+            <th class="col-lg-2" translate>mnoe_admin_panel.dashboard.dashboard_templates.widget.list.updated</th>
+            <th class="col-lg-2"></th>
+          </tr>
+<!--           <tr>
+            <th>
+              <input st-search="name" placeholder="{{'mnoe_admin_panel.dashboard.dashboard_templates.widget.list.search' | translate}}" class="form-control input-sm search-bar" type="text"/>
+            </th>
+            <th></th>
+            <th></th>
+            <th></th>
+            <th></th>
+          </tr> -->
+        </thead>
+        <tbody >
+          <tr ng-repeat="dashboardTemplate in $ctrl.dashboardTemplates.list">
+            <td ng-show="!dashboardTemplate.editMode" title="{{ dashboardTemplate.full_name }}">{{ dashboardTemplate.name }}</td>
+            <td ng-show="dashboardTemplate.editMode">
+              <input placeholder="{{ dashboardTemplate.name }}" ng-model="dashboardTemplate.newName" class="form-control input-sm" type="text"/>
+            </td>
+            <td>{{ dashboardTemplate.status || "published"}}</td>
```

Comment:
This will be returned by the back-end and I'm not sure yet how to handle it.
Storing a locale key in the back will make the code complex as tests are going to be performed on the value of `dashboard.status`...

I sugggest we simply put `<td>{{ dashboardTemplate.status }}</td>` for now - ok?